docs/examples/DataSet/Using_the_doNd_functions_for_running_experiments.ipynb

```diff
@@ -225,79 +439,92 @@
     }
    ],
    "source": [
+    "# Creating an experiment\n",
+    "exp4 = load_or_create_experiment('2D_do2d', sample_name='no sample')\n",
+    "\n",
+    "# Running masurement with do2d\n",
     "result = do2d(dac.ch1, -1, 1, 20, 0.01, dac.ch2, -1, 1, 20, 0.01, dmm.v1, dmm.v2, do_plot = True)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that `do_plot = True` is the default value and we only added it for the sake of explicitness. The `doNd` utility, in fact, uses an internal public function called `plot` to handle data visualization, which by default saves the figures in both pdf and png formats, as well. Therefore, users, if needed, can only do their measurements with the option `do_plot = False` and then pass the relevant dataset to `plot` when visualization is necessary and choose to save figures in a desired format.\n",
+    "### Handling plot and experiment with doNd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `doNd` utility, in fact, uses an internal public function called `plot` to handle data visualization, which by default saves the figures in both pdf and png formats, as well. Therefore, users, if needed, can only do their measurements leaving to set `do_plot`, which by default is False, then pass the relevant dataset to `plot` when visualization is necessary and choose to save figures in a desired format.\n",
     "\n",
-    "Internally, the `doNd` functions return a tuple consists of the dataset obtained in the measurement, the boolean value correponding to `do_plot` and a callable which checks whether the experiment is interupted by the user. Thus, if the `plot` shall be used explicitly after the measurement, one has to pass the first argument of the return value. "
+    "Internally, the `doNd` functions return a tuple consists of the dataset obtained in the measurement, the boolean value correponding to `do_plot` and a callable which checks whether the experiment is interupted by the user. Thus, if the `plot` shall be used explicitly after the measurement, one has to pass the first argument of the return value.\n",
+    "\n",
+    "Moreover, it is possible to pass an already existing experiment as an argument to `doNd` constructor. This is usually a convenient way as users may prefer not to create a new experiment before running a measurement with `doNd`s. "
```

Comment:
Suggest something like

"
As with the Measurement context manager it is possible to pass an explicit experiment to the `doNd` functions. This makes it possible to switch between experiments when using the `doNd` functions